app/models/concerns/feedback_validations.rb

```diff
@@ -0,0 +1,12 @@
+module FeedbackValidations
+  extend ActiveSupport::Concern
+
+  included do
+    validates :user_participation_response, presence: true
+    validates :email, email_address: { presence: true }, if: :user_is_interested?
+  end
+
+  def user_is_interested?
+    return true if user_participation_response == 'interested'
```

Comment:
Minor, but could be simplified to `return user_participation_response == 'interested'`